pkgs/applications/blockchains/feather-wallet/default.nix

```diff
@@ -0,0 +1,84 @@
+{ lib, stdenv, wrapQtAppsHook, makeDesktopItem
+, fetchFromGitHub, fetchgit
+, git, cmake, qttools, pkg-config
+, qtbase, qtwebsockets, qtsvg
+, miniupnpc, unbound, readline
+, boost, libunwind, libsodium, pcsclite
+, randomx, zeromq, libgcrypt, libgpg-error
+, hidapi, rapidjson, qrencode, zbar, expat
+, libzip, libusb1, protobuf, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "feather-wallet";
+  version = "2.1.0";
+
+  src = fetchgit {
```

Comment:
Consider using function `fetchFromGitHub`

Reply:
The build process needs the `.git` directory to get version information, so I had to use `fetchgit` with the `leaveDotGit = true` option. I don't know if `fetchFromGitHub` has this option.

Reply:
`leaveDotGit` should also work for `fetchFromGitHub` https://github.com/NixOS/nixpkgs/blob/350fd0044447ae8712392c6b212a18bdf2433e71/pkgs/applications/virtualization/ignite/default.nix#L21 
Would be cool to use this!